hadoop-hdds/common/src/main/java/org/apache/hadoop/ozone/lease/LeaseManager.java

```diff
@@ -63,19 +63,19 @@ public class LeaseManager<T> {
    *        Default timeout in milliseconds to be used for lease creation.
    */
   public LeaseManager(String name, long defaultTimeout) {
-    this.name = name;
+    this.name = name + "LeaseManager";
```

Comment:
I should be "-LeaseManager", instead of "LeaseManager".

Reply:
`name` in `name + "LeaseManager"` comes from the SCM-specific prefix.  Which means it has the node info, and it ends in `-`.  So `this.name` ends up like `scmNode-1-LeaseManager`.  The monitor thread is then named `scmNode-1-LeaseManager#LeaseMonitor`.